gammapy/data/ivoa.py

```diff
@@ -0,0 +1,327 @@
+        dtype="U30",
+        meta={"Utype": "Char.ObservableAxis.ucd"},
+    )
+    obscore_default[26] = Column(
+        name="pol_xel",
+        unit="",
+        description="Number of elements along the polarization axis",
+        dtype="i4",
+        meta={"Utype": "Char.PolarizationAxis.numBins"},
+    )
+    obscore_default[27] = Column(
+        name="facility_name",
+        unit="",
+        description="The name of the facility, telescope space craft used for the observation",
+        dtype="U10",
+        meta={"Utype": "Provenance.ObsConfig.Facility.name"},
+    )
+    obscore_default[28] = Column(
+        name="instrument_name",
+        unit="",
+        description="The name of the instrument used for the observation",
+        dtype="U25",
+        meta={"Utype": "Provenance.ObsConfig.Instrument.name"},
+    )
+    tab_default = Table()
+    for var in obscore_default:
+        tab_default.add_column(var)
+    return tab_default
+
+
+def _obscore_row(base_dir, single_obsID, obs_publisher_did, access_url, table):
+    """Generates an obscore row corresponding to a single obsID
+
+    Parameters
+    ----------
+    single_obsID : int
+        single Observation ID
+    **kwargs : `str` {obs_publisher_did, access_url}
+    Giving the values for is highly recommended.
+    If any of these are not given the corresponding obscore field is left empty and a warning is raised for each empty value.
+
+    Returns
+    -------
+    table : `~astropy.table.Table`
+    """
+    base_dir = make_path(base_dir)
+    data_store = DataStore.from_dir(base_dir)
+
+    if obs_publisher_did is None:
+
+        log.warning(
+            "Insufficient publisher information: 'obs_publisher_did' obscore value will be empty."
+        )
+    if access_url is None:
+        log.warning(
+            "Insufficient publisher information: access_url' obscore value will be empty."
+        )
+
+    tab = table
+    observation = data_store.obs(single_obsID)
+    obs_mask = data_store.obs_table["OBS_ID"] == observation.obs_id
+    obs_pos = data_store.obs_table[obs_mask]["EVENTS_FILENAME"]
+    path = make_path(os.environ["GAMMAPY_DATA"] + "/" + obs_pos[0])
+    size = int(os.path.getsize(path) / 1000.0)
+    tab.add_row(
+        {
+            "dataproduct_type": observation.obs_info["EXTNAME"],
+            "calib_level": 2,  # Look into the data
+            "target_name": observation.obs_info["OBJECT"],
+            "obs_id": str(observation.obs_info["OBS_ID"]),
+            "obs_collection": "DL3",
+            "obs_publisher_did": str(obs_publisher_did),
+            "access_url": str(access_url),
+            "access_format": "application/fits",
+            "access_estsize": size,
+            "s_ra": observation.get_pointing_icrs(observation.tmid).ra.to_value("deg"),
+            "s_dec": observation.get_pointing_icrs(observation.tmid).dec.to_value(
+                "deg"
+            ),
+            "s_fov": 10.0,
```

Comment:
The FoV is not fixed. It should be guessed from the IRFs?

Reply:
Yes, I agree, but we have not yet defined how to do it, so for now it is fixed. If you agree we can add a comment on this line.